tests/es/test_client.py

```diff
@@ -150,3 +154,32 @@ async def does_not_raise():
     await does_not_raise()
 
     assert mock_func.call_count == 1
+
+
+@pytest.mark.parametrize(
+    "enabled_license, licenses_enabled",
+    [
+        (License.TRIAL, [License.TRIAL]),
+        (License.BASIC, [License.TRIAL, License.BASIC]),
+        (License.PLATINUM, [License.TRIAL, License.BASIC, License.PLATINUM]),
+        (License.ENTERPRISE, [License.TRIAL, License.BASIC, License.PLATINUM, License.ENTERPRISE])
```

Comment:
I don't think this is quite right. A "trial" license entitles you to everything an Enterprise license does. Basic is actually the least-privilaged.